src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -1,9 +1,543 @@
+    clearDetails,
+    updateFacilityName,
+    updateFacilityAddress,
+    updateFacilityCountry,
+    updateFacilityPhone,
+    updateFacilityWebsite,
+    updateFacilityDescription,
+    updateFacilityMinimumOrder,
+    updateFacilityAverageLeadTime,
+    updateContactPerson,
+    updateContactEmail,
+    updateOfficeName,
+    updateOfficeAddress,
+    updateOfficeCountry,
+    updateOfficePhone,
+    submitUpdate,
+    updating,
+    updateFacilityPhoneVisibility,
+    updateContactVisibility,
+    updateOfficeVisibility,
+    errorUpdating,
+}) {
+    /* eslint-disable react-hooks/exhaustive-deps */
+    // disabled because we want to use this as just
+    // componentDidMount and componentWillUpdate and declaring the
+    // methods in the array here caused an infinite loop for some reason
+    useEffect(() => {
+        getDetails();
+
+        return clearDetails;
+    }, []);
+    /* eslint-enable react-hooks/exhaustive-deps */
+
+    if (fetching) {
+        return <CircularProgress />;
+    }
+
+    if (error) {
+        return (
+            <Typography variant="body1">
+                An error prevented fetching that facility claim.
+            </Typography>
+        );
+    }
+
+    if (!data) {
+        return null;
+    }
+
+    const countryOptions = createCountrySelectOptions(data.countries);
+
+    return (
+        <div style={claimedFacilitiesDetailsStyles.containerStyles}>
+            <div style={claimedFacilitiesDetailsStyles.formStyles}>
+                <Typography
+                    variant="title"
+                    style={claimedFacilitiesDetailsStyles.headingStyles}
+                >
+                    Facility Details
+                </Typography>
+                <InputSection
+                    label="Facility name"
+                    value={data.facility_name}
+                    onChange={updateFacilityName}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Address"
+                    value={data.facility_address}
+                    onChange={updateFacilityAddress}
+                    disabled={updating}
+                />
+                <InputSection
+                    label="Country"
+                    value={data.facility_country}
+                    onChange={updateFacilityCountry}
+                    disabled={updating}
+                    isSelect
+                    selectOptions={countryOptions}
+                />
+                <InputSection
+                    label="Phone Number"
+                    value={data.facility_phone_number}
+                    onChange={updateFacilityPhone}
+                    disabled={updating}
+                    hasSwitch
+                    switchValue={data.facility_phone_number_publicly_visible}
+                    onSwitchChange={updateFacilityPhoneVisibility}
+                />
+                <InputSection
+                    label="Website"
```

Comment:
Consider creating a follow up issue to add url validation to this field and to render it as an active link in the sidebar (using the same conditional "http://" appending that we use on the contributor profile page.

Reply:
This is/will be https://github.com/open-apparel-registry/open-apparel-registry/issues/587